### PR TITLE
got to the psudo code and couldnt figure out how to write it.

### PR DESCRIPTION
**Pull request by M-Ravenmoore**
- [x] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/multiBracketValidation/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+then resolve the stack looking for close brackets starting from the index of the top bracket on the stack.
+
+ if the stack clears return true
+
+else return false
+
+## Solution
+<!-- Embedded whiteboard image -->
+![whiteboard](../assets/CC13-whiteboard.jpg)
```
Comment (adrianhuebner) [inline javascript/code-challenges/multiBracketValidation/README.md:23]: You're off to a great start. Would just like to see some code and tests added.
